web/src/app/shared/host-components-map/much-2-many/much-2-many.component.ts

```diff
@@ -37,10 +42,21 @@ export class Much2ManyComponent {
     if ('service_id' in this.model && Object.keys(this.form.controls).length) {
       const sc = this.model as CompTile;
       const control = this.form.controls[`${sc.service_id}/${sc.id}`];
-      if (!control) return false;
-      sc.notification = control.errors?.error;
-      return control.invalid;
-    } else return false;
+
+      if (!control || !control.errors?.error) return true;
```

Comment:
may be change to `!control!.errors?.error`?

Reply:
won't hurt, fixed